### DISCUSSION TITLE
ANTsPy: Registration, how to code convergence value ?

### DISCUSSION THREAD

**Discussion by flo-mingo** [ANTsX/ANTs#1769] [Q&A]
Hello!

I'm looking for a way to add a convergence value in my registration parameters. Could it be in kwargs ?  or somewhere else ? 
You can see below an exemple of my transform, and I would like to set the convergence value to 0, but couldn't find a way. 
Thanks !

`mytx = ants.registration(fixed=MRI_ants_mask, moving=IHC_ants_mask, type_of_transform='Affine', aff_sampling=2, metric_convergence_threshold=0,
                             aff_iterations=(200, 100, 100, 20), aff_shrink_factors=(6, 4, 2, 1), aff_smoothing_sigmas=(3, 2, 1, 0), verbose=True)`

**Comment by cookpa** [comment]
Unfortunately, I don't think there's any way to set this in `ants.registration`
